fetch.bs

```diff
@@ -6011,7 +6132,8 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
 
    <li><p>If <var>connection</var> is an HTTP/1.x connection, <var>request</var>'s
    <a for=request>body</a> is non-null, and <var>request</var>'s <a for=request>body</a>'s
-   <a for=body>source</a> is null, then return a <a>network error</a>.
+   <a for=body>source</a> is null, then return the result of running <a>HTTPS upgrade fallback</a>
+   given <var>request</var> and <var>network error</var>.
```

Comment:
This is a streaming case. I don't think we should touch this. This also won't be true for the scenario we care about: navigation.